### ISSUE TITLE
OCaml 4.06.0 Support

### ISSUE THREAD

**Issue by shinzui** [ocaml-toml/To.ml#52]
Is this package compatible with 4.06.0? I get an error when installing through opam.

```
toml is not available because your system doesn't comply with ocaml-version >= "4.01.0" & ocaml-version < "4.06.0".
```

**Comment by orbifx** [comment]
I think the version in OPAM is outdated. Try pinning the master from here:

    opam pin add toml https://github.com/orbifx/To.ml.git

**Comment by shinzui** [comment]
@orbifx I'm using the latest stable opam version (1.2.2).

I did pin it to fix my problem but I opened the issue to help others.

**Comment by basus** [comment]
Can we get this fixed please? If there is actual work to be done to make it compatible with OCaml 4.06.0, I'm happy to do it and submit a pull request, if I'm told what it is. If it's just a question of updating a file or setting somewhere, can the maintainer (I'm guessing it's @mackwic) do it soon?

**Comment by c-cube** [comment]
should be fixed now with 5.0, please re-open if there are issues.
